lab_5_scrapper/scrapper.py

```diff
@@ -120,19 +224,38 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        paragraphs = article_soup.find_all('div', class_="news-detail hyphenate")
+        links = [tag.text for tag in paragraphs]
+        for link in links:
+            new_l = link.split('.')
+            self.article.text = '.'.join(new_l[:-3])
```

Comment:
The fact is that the text of the article on my site is enclosed in one single tag. There is no <p> tags, all paragraphs are just strings separated by <br> tag. I've checked, this works and all the paragraphs are stored.

Reply:
It is not clear from your code. What happens is that you assign a completely new value to the `self.article.text` as many times as there are `link`s in `links`. If there is only one element in the `links` sequence expected, this should be reflected in your code. Otherwise, if your expectations happen to be wrong, you'll have a hard time tracing this bug.